src/components/Grid.astro

```diff
@@ -0,0 +1,4 @@
+<div>
+<div class="not-prose flex flex-col md:flex-row pt-4 md:space-x-6 lg:space-x-12 pb-16 space-y-6 md:space-y-0">
```

Comment:
You should use the grid classes for this, it's much easier to read and more true to what it's doing.
If there are 10 children, you don't want them all on the same line.

https://tailwindcss.com/docs/grid-template-columns

Reply:
This has been implemented